cmd/centrifuge/root.go

```diff
@@ -39,6 +41,12 @@ func Execute() {
 	}
 }
 
+func printStartMessage() {
+	c := figure.NewFigure("Centrifuge OS", "", true)
+	c.Print()
+	fmt.Println("Centrifuge is beta software !!!")
```

Comment:
Do we need to add more information about what this beta version means?

Reply:
I also thought of this, but for the console its too much information. I think we can explain that in the disclaimer on develop website.